code/modules/mob/living/carbon/human/life.dm

```diff
@@ -671,15 +671,24 @@
 	breath.update_values()
 	return 1
 
-/mob/living/carbon/human/handle_environment(datum/gas_mixture/environment)
-	if(!environment)
-		return
+/**
+ * @params
+ * * environment - the gas mixture we're in
+ * * dt - seconds to simulate
+ */
+/mob/living/carbon/human/handle_environment(datum/gas_mixture/environment, dt)
```

Comment:
Please just use DeltaTime instead of dt, who know's it could be Lohikar's abbreviation of dmt!

Reply:
In hindsight I don't care to much, just wanted to point it out

Reply:
we should regex it later to delta_time or seconds or seconds_tick or something